src/frontends/onnx/frontend/src/ops_bridge.hpp

```diff
@@ -80,26 +87,11 @@ class OperatorsBridge {
     //    domain_2: { ... },
     //    ...
     // }
-    std::unordered_map<std::string, std::unordered_map<std::string, std::map<std::int64_t, Operator>>> m_map;
-
-    OperatorsBridge();
-
-    static OperatorsBridge& instance() {
-        static OperatorsBridge instance;
-        return instance;
-    }
-
-    void _register_operator(const std::string& name, std::int64_t version, const std::string& domain, Operator fn);
-    void _unregister_operator(const std::string& name, std::int64_t version, const std::string& domain);
-    OperatorSet _get_operator_set(const std::string& domain, std::int64_t version);
-
-    bool _is_operator_registered(const std::string& name, std::int64_t version, const std::string& domain);
-
-    void _load_initial_state();
-    std::mutex lock;
+    using DomainOpset = std::unordered_map<std::string, std::map<std::int64_t, std::shared_ptr<Operator>>>;
+    std::unordered_map<std::string, DomainOpset> m_map;
 };
 
-const std::string OPENVINO_ONNX_DOMAIN = "org.openvinotoolkit";
+static constexpr const char* OPENVINO_ONNX_DOMAIN = "org.openvinotoolkit";
```

Comment:
In fact I have to leave it as `constexpr const char* OPENVINO_ONNX_DOMAIN = "org.openvinotoolkit"`. Otherwise some tests fail because the string is not initialized correctly here (in C++11 its constructor is not constexpr).
Additionally this doesn't compile:
`constexpr char* OPENVINO_ONNX_DOMAIN = "org.openvinotoolkit";`

Reply:
Right .. `const` applies to pointed memory, not the variable itself.